examples/io_test/io_plane_test.f90

```diff
@@ -1,9 +1,15 @@
+!! NB in case of GPU only the writing in the aligned pencil (i.e. X for a 1 array) is performed. 
+!! IO subrotines needs update for non managed GPU case
```

Comment:
We should write known issues in a dedicated file ? The comments in a example is probably not the best location.

Reply:
I add this note in the file to explain why i put an `#ifdef ` around some IO options. We should fix the IO routine but I before we need to agree what we want to do with IO and GPU.

Reply:
In my opinion, we should not modify the example. I suggest we have a dedicated issue and the following  : 

- Keep the current `call decomp_2d_write_plane` in the example
- Update the IO subroutines when writing plane

In case of GPU build, when writing planes, if transpose operations are needed and not supported, one can : 

- Print some error message and abort the simulation
- Print some warning message, skip the IO and run the simulation
- Silently skip the IO and run the simulation

Reply:
What i have is already skipping the IO that it is not supported. I’ll add a warning. To fix the IO i would like to first check with Paul.

Reply:
My apologies if my previous comment was not clear. In my opinion, the tests performed in `io_plane_test.f90` should not be skipped for a GPU build.

The file `io.f90` or the file `io_write_plane.inc` should be updated instead. If we decide to print some error and abort, the modification should be

```
#if defined(_GPU)
   if (ipencil /= iplane) call decomp_2d_abort(__FILE__, &
                                                                               __LINE__, &
                                                                               ipencil, &
                                                                             "GPU build does not allow transpose operations when writing planes")
#endif
```

If we decide to print some warning and skip IO, the modification should be

```
#if defined(_GPU)
   if (ipencil /= iplane) then
      call decomp_2d_warning(__FILE__, &
                                                    __LINE__, &
                                                    ipencil, &
                                                    "GPU build does not allow transpose operations when writing planes")
      return
   endif
#endif
```

Reply:
Yes we should update the IO files, however i think we need to check with Paul before going on modifying the files. I know how to fix the problems on the IO file, just need to check how this is fitting in the overall development for the IO.

Reply:
Ok i think we can go for the second option of a warning and a return. I think the test needs to be completed since this is not really a bug but just a feature not being up to date. 
I’ll use the second option.

Reply:
I have pushed the warning + skip IO update. Only thing needed to resolve the conversation is to open a dedicated issue and clean the comments in examples/io_test/io_plane_test.f90

Reply:
Please keep in mind that the issue of plane IO for GPU builds can have a dedicated PR, it is not mandatory to solve it in this PR